File: lib/py/uapi/internal/generation/GenerateRandomAny.py
from typing import TYPE_CHECKING


if TYPE_CHECKING:
    from uapi.internal.generation.GenerateContext import GenerateContext
    from uapi.RandomGenerator import RandomGenerator


def generate_random_any(ctx: 'GenerateContext') -> object:
    select_type = ctx.random_generator.next_int_with_ceiling(3)
    if select_type == 0:
        return ctx.random_generator.next_boolean()
    elif select_type == 1:
        return ctx.random_generator.next_int()
    else:
        return ctx.random_generator.next_string()

File: lib/ts/uapi/internal/generation/GenerateRandomAny.ts
import { RandomGenerator } from 'uapi/RandomGenerator';

export function generateRandomAny(randomGenerator: RandomGenerator): any {
    const selectType = randomGenerator.nextIntWithCeiling(3);
    if (selectType === 0) {
        return randomGenerator.nextBoolean();
    } else if (selectType === 1) {
        return randomGenerator.nextInt();
    } else {
        return randomGenerator.nextString();
    }
}
